integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -285,19 +298,19 @@ protected String readGitCommitIdFromGitFile() {
     }
   }
 
-  private static boolean isGravitinoServerUp() {
-    JettyServerConfig jettyServerConfig =
-        JettyServerConfig.fromConfig(serverConfig, WEBSERVER_CONF_PREFIX);
-    String host = jettyServerConfig.getHost();
-    int port = jettyServerConfig.getHttpPort();
-    int timeout = 3000; // 3 second timeout
-
-    try (Socket socket = new Socket()) {
-      socket.connect(new java.net.InetSocketAddress(host, port), timeout);
-      LOG.info("Gravitino Server is up and running.");
-      return true;
+  public static boolean isGravitinoServerUp(String url) {
```

Comment:
Could you make it more general to be reused be Iceberg REST service? like rename to `isServerUp` and check HTTP status is 200 ?

Reply:
OK, it seems that checking the response code is sufficient.